analyze_nplus_latest_directives.go

```diff
@@ -2151,7 +2151,7 @@ var ngxPlusLatestDirectives = map[string][]uint{
 	},
 }
 
-func NgxPlusLatestDirectivesMatchFn(directive string) ([]uint, bool) {
+func MatchLatestNgxPlus(directive string) ([]uint, bool) {
```

Comment:
Can you add doc comments for the remaining match functions? If that's a different PR, that's OK too.

Reply:
Also this should be `MatchNginxPlusLatest`

- Don't abbreviate "Nginx". The name is short enough as it is. We're not gaining anything and losing clarity.
- Follow the naming pattern esablished (`MatchModuleVersion`) so that the identifiers sort well. If you transpose the module name and version then the list looks unsorted and harder to scan in code editors as well as [pkg.go.dev](https://pkg.go.dev/github.com/nginxinc/nginx-go-crossplane)

Reply:
Yeah since code comments will be included in `go generate`, I plan to add the corresponding comments when we have `go generate` comment for them.

Reply:
Applied: `Ngx` to `Nginx` in directive map variable name and MatchFunc, rename modules matchfunc to follow `MatchModuleVersion` pattern